apps/blockchain/lib/blockchain/block.ex

```diff
@@ -230,6 +230,54 @@ defmodule Blockchain.Block do
     end
   end
 
+  @doc """
+  Returns the specified block from the database, if the block number is found in the provided map
+  and the resulting block hash is found in the provided TrieStorage.
+
+  ## Examples
```

Comment:
hm... should we move these into block_test.exs? makes more sense as the module gets bigger I think